jaxrs-api/src/main/java/jakarta/ws/rs/core/Form.java

```diff
@@ -93,4 +93,29 @@ public Form param(final String name, final String value) {
     public MultivaluedMap<String, String> asMap() {
         return parameters;
     }
+
+    /**
+     * Compare for equality
+     * 
+     * @param object the object to compare to.
+     * @return {@code true}, if the object is a {@code Form} with the same form parameters, {@code false} otherwise.
+     */
+    @Override
+    public boolean equals(Object object) {
+        if (object == null) return false;
+        if (object == this) return true;
+        if (getClass() != object.getClass()) return false;
+        Form form = (Form) object;
+        return parameters.equals(form.parameters);
+    }
+
+    /**
+     * Returns the hashCode of the underlying form parameters.
+     * 
+     * @return a hash code value for the underlying form parameters.
+     */
+    @Override
+    public int hashCode() {
+        return parameters.hashCode();
```

Comment:
IMO we should use something like `Objects.hash(parameters)`.

Reply:
What would be the benefit over the current draft?

Reply:
It's not critical by any means, it's just not the standard pattern to not calculate a hash like this. I suppose that's up to opinion though :) This should not block this FWIW.